dom.css

```diff
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  width: 100%;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
margin left and margin right are not required when height and width are set to 100%.